scripts/backup-authored-files.sh

```
@@ -0,0 +1,80 @@
+    echo "error: $f did not transfer intact (want ${want:-?}, got ${got:-nothing})" >&2
+    failed="$failed $f"
+  fi
+done
+
+if [ -n "$failed" ]; then
+  echo "backup INCOMPLETE, keeping nothing:$failed" >&2
+  exit 1
+fi
+
+archive="$DEST/beeline-authored-$stamp.tar.gz"
+tar -czf "$archive" -C "$work" $FILES
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)